templates/repo/release/list.tmpl

```diff
@@ -53,21 +53,28 @@
 							<div class="download">
 								<h2>{{$.i18n.Tr "repo.release.downloads"}}</h2>
 								<ul class="list">
+									{{$tagname := .TagName}}
```

Comment:
Now that you have added $release you can use $release.TagName and this variable is not needed anymore